tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -36,11 +36,10 @@ def test_datafile(stt_user, stt):
 @pytest.mark.django_db
 def test_parse_small_correct_file(test_datafile):
     """Test parsing of small_correct_file."""
-    errors = parse.parse_datafile(test_datafile)
+    parse.parse_datafile(test_datafile)
```

Comment:
Can you revert the errors here? It will eventually be used in 1613, same with `assert errors == {}` to make linter happy.

Reply:
this is a miss from working late last night 👍  Will revert it back for sure